Description: Fix upstream code. Reflect the new path.
Author: Christophe Mutricy <xtophe@videolan.org>
Applied-Upstream: 1.1.1

Index: debian.git/modules/misc/notify/notify.c
===================================================================
--- debian.git.orig/modules/misc/notify/notify.c	2010-04-13 02:22:27.000000000 +0200
+++ debian.git/modules/misc/notify/notify.c	2010-06-23 00:46:58.104227185 +0200
@@ -225,14 +225,7 @@
     else /* else we show state-of-the art logo */
     {
         GError *p_error = NULL;
-        char *psz_pixbuf;
-        char *psz_data = config_GetDataDir( p_this );
-        if( asprintf( &psz_pixbuf, "%s/vlc48x48.png", psz_data ) >= 0 )
-        {
-            pix = gdk_pixbuf_new_from_file( psz_pixbuf, &p_error );
-            free( psz_pixbuf );
-        }
-        free( psz_data );
+        pix = gdk_pixbuf_new_from_file( "/usr/share/icons/hicolor/48x48/apps/vlc.png", &p_error );
     }
 
     free( psz_arturl );
